daemon/logs.go

```diff
@@ -79,11 +79,16 @@ func (daemon *Daemon) ContainerLogs(containerName string, config *backend.Contai
 			logrus.Errorf("Error streaming logs: %v", err)
 			return nil
 		case <-config.Stop:
-			logs.Close()
+			if !container.IsRunning() {
```

Comment:
@calavera Looking at it again, I realize that <code>logs</code> is not a Logger, but actually a LogWatcher.  Given that, I think the right behavior is to always call <code>Close()</code> in both cases, regardless of whether the container is running.  I'll update the code and try that solution.